Back/src/utils/users/mod.rs

```diff
@@ -0,0 +1,36 @@
+use std::{fs, path::PathBuf};
+
+use crate::database::{model::User, UserData};
+
+pub struct VerifiedUserPath {
+    _user: User,
```

Comment:
In rust `_` don't mean the same thing as in cpp
We already have the prefix pub for public/private members
_ is almost exclusively used to describe unused variables or match branch